ui/partials/perf/revisiondescribe.html

```diff
@@ -0,0 +1,12 @@
+<li>
+    <strong>Base</strong> - <a href="{{originalProject.getRevisionHref(originalRevision)}}">{{originalRevision}}</a> ({{originalProject.name}}) - {{originalResultSet.author}} -
+    <span>{{originalResultSet.comments.substring(0, 60)}}</span>
```

Comment:
After testing, I don't think the show/hide behaviour is particularly useful here: I think this logic makes more sense in the main treeherder view, where space is much more limited.. I think we might as well just show the whole thing by default.

So just replace lines 3-5 with: {{originalResultSet.comments}} (no need for a containing span)